containers/rust-postgres/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# [Choice] Debian OS version (use bullseye on local arm64/Apple Silicon): buster, bullseye
+ARG VARIANT="bullseye"
+FROM mcr.microsoft.com/vscode/devcontainers/rust:1-${VARIANT}
+
+# Add lld linker
```

Comment:
Could you add a sentence in the Dockerfile explaining why you are adding the `lld linker` (for other future maintainers of the repo to reference)?